<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QWidget" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>426</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Lobby</string>
  </property>
  <widget class="QLabel" name="lbTitolo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>571</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UPWORDS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="swPage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>521</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="pageOne">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>501</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btUnGiocatore">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>UN GIOCATORE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btDueGiocatori">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>DUE GIOCATORI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btTreGiocatori">
        <property name="text">
         <string>TRE GIOCATORI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btQuattroGiocatori">
        <property name="text">
         <string>QUATTRO GIOCATORI</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="pageTwo">
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>491</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbGiocatoreUno">
        <property name="text">
         <string>Giocatore Uno</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txGiocatoreUno"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbGiocatoreDue">
        <property name="text">
         <string>Giocatore Due</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txGiocatoreDue"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbGiocatoreTre">
        <property name="text">
         <string>Giocatore Tre</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txGiocatoreTre"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbGiocatoreQuattro">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Giocatore Quattro:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txGiocatoreQuattro"/>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="lbErrore">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="btStart">
        <property name="text">
         <string>START</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btIndietro">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
